docs/observability/getting-started.md

```diff
@@ -0,0 +1,89 @@
+
+Setting up Genkit Monitoring requires completing tasks in both your codebase and on the Google Cloud Console.
+
+## Before you begin
+
+1. If you haven't already, create a Firebase project. 
+
+    In the [Firebase console](https://console.firebase.google.com), click **Add a project**, then follow the on-screen instructions. You can create a new project or add Firebase services to an already existing Google Cloud project.
+2. Ensure your project is on the [Blaze pricing plan](https://firebase.google.com/pricing). 
+
+    Genkit Monitoring relies on telemetry data written to Google Cloud Logging, Metrics, and Trace which are paid services. View the [Google Cloud Observability](http://cloud/stackdriver/pricing#cloud-monitoring-pricing) page for pricing details and to learn about available free tier limits.
+3. Write a Genkit feature by following the [Get Started Guide](https://firebase.google.com/docs/genkit/get-started) and prepare your code for deployment by using one of the following guides:
+    1. [Deploy with Firebase](../firebase.md)
+    2. [Deploy with Cloud Run](../cloud-run.md)
+    3. [Deploy to any Node.js platform](../deploy-node.md)
+
+
+## Step 1. Add the Firebase plugin
+
+Install the `@genkit-ai/firebase` plugin in your project:
+
+```
+npm i –save @genkit-ai/firebase
+```
+
+Import `enableFirebaseTelemetry` into your Genkit configuration file (i.e. where `genkit(...)` is initalized) and call it:
+
+```typescript
+enableFirebaseTelemetry({
+  forceDevExport: false // set to true for local testing
+});
+```
+
+## Step 2. Enable the required APIs
+
+Make sure that the following APIs are enabled for your GCP project:
+
+* [Cloud Logging API](https://console.cloud.google.com/apis/library/logging.googleapis.com)
+* [Cloud Trace API](https://console.cloud.google.com/apis/library/cloudtrace.googleapis.com)
+* [Cloud Monitoring API](https://console.cloud.google.com/apis/library/monitoring.googleapis.com)
+
+These APIs should be listed in the [API dashboard](https://console.cloud.google.com/apis/dashboard) for your project.
+
+## Step 3. Set up permissions
+
+The Firebase plugin needs to authenticate with Google Cloud Logging, Metrics, and Trace services using a _service account_.
+
+Grant the following roles to whichever service account is configured to run your code within the [Google Cloud IAM Console](https://console.cloud.google.com/iam-admin/iam). For Firebase Functions and/or Cloud Run, this is typically the “default compute service account”.
+
+* **Monitoring Metric Writer** (`roles/monitoring.metricWriter`)
+* **Cloud Trace Agent** (`roles/cloudtrace.agent`)
+* **Logs Writer** (`roles/logging.logWriter`)
+
+## Step 4. (Optional) Test your configuration locally
+
+Before deploying, you can run your Genkit code locally to confirm that telemetry data is being collected and viewable in the Genkit Monitoring dashboard.
+
+1. In your Genkit code, set `forceDevExport` to `true` to send telemetry from your local environment.
```

Comment:
We should probably also mention that when running locally they may want to increase the metric export interval so that they can see metrics faster. By default we set this to 5min, but it can be reduced to as low as 5 seconds.

Maybe we just put a copiable code block?

Reply:
@MichaelDoyle  maybe line 82 instead of "(based on the default `metricExportIntervalMillis` setting in the telemetry configuration)" we could say something like "this timing can be reduced by setting the `metricExportIntervalMillis` property in the Firebase telemetry configuration)"